### PR TITLE
Make linter working with Python3

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

File: third_party/cpplint.py [hunk 1]
```diff
@@ -5553,7 +5555,7 @@ def CheckForIncludeWhatYouUse(filename, clean_lines, include_state, error,
 
   # include_dict is modified during iteration, so we iterate over a copy of
   # the keys.
-  header_keys = include_dict.keys()
+  header_keys = [k for k in include_dict.keys()]
```
Comment (klecki) [inline third_party/cpplint.py]: Can't you just `list(include_dict.keys())`?

Comment (JanuszL) [reply third_party/cpplint.py]: done

**Comment by JanuszL** [comment]
!build

**Pull request merged**